<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AutoUpdate</class>
 <widget class="QWidget" name="AutoUpdate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AutoUpdate</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="3" column="0">
    <widget class="QPushButton" name="btnUpdate">
     <property name="text">
      <string>Update</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBoxDescription">
     <property name="title">
      <string>Description</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QTextBrowser" name="textBrowserDescription"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBoxUpdate">
     <property name="title">
      <string>Update</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="radioButtonExeOnly">
        <property name="text">
         <string>Exe Only</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButtonFull">
        <property name="text">
         <string>Full</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBoxVersion">
     <property name="title">
      <string>Version</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="labelURL">
        <property name="text">
         <string>URL</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QLineEdit" name="lineEditURL"/>
      </item>
      <item row="1" column="2">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="hLayTargetVersion">
          <item>
           <widget class="QLabel" name="labelTargetVersion">
            <property name="text">
             <string>Target Version</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditTargetVersion"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="labelTimestamp">
          <property name="text">
           <string>Timestamp</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0" colspan="2">
       <layout class="QHBoxLayout" name="hLayCurrentVersion">
        <item>
         <widget class="QLabel" name="labelCurrentVersion">
          <property name="text">
           <string>Current Version</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditCurrentVersion"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
